subprojects/docs/src/docs/design/gradle-module-metadata-1.0-specification.md

```diff
@@ -251,13 +246,6 @@ This value, nested in `variants`, must contain an array with zero or more elemen
                     "module": "other-lib", 
                     "version": { "requires": "[3.0, 4.0)", "prefers": "3.4", "rejects": ["3.4.1"] } 
                 }
-            ],
-            "dependencyConstraints": [
-                { 
-                    "group": "some.group", 
-                    "module": "other-lib-2", 
-                    "version": { "requires": "1.0", "forSubgraph": true } 
-                }
```

Comment:
We could leave this in here (only without the `"forSubgraph": true `). `"dependencyConstraints"` where missing in the example before.